docs/alerts/scheduled-searches/create-real-time-alert.md

```diff
@@ -48,7 +48,7 @@ The results from your search will vary based on the type of alert selected. The
 
 | Alert Type | Results in Notification |
 | :-- | :-- |
-| [Webhook](/docs/manage/connections-integrations/webhook-connections/schedule-searches-webhook-connections.md) | If the **Send a separate alert for each search result checkbox** is selected (in step 6) only new results from subsequent searches are sent in the alert payload. Otherwise, all results are sent. |
+| [Webhook](/docs/alerts/webhook-connections/schedule-searches-webhook-connections) | If the **Send a separate alert for each search result checkbox** is selected (in step 6) only new results from subsequent searches are sent in the alert payload. Otherwise, all results are sent. |
```

Comment:
```suggestion
| [Webhook](/docs/alerts/webhook-connections/schedule-searches-webhook-connections) | If the **Send a separate alert for each search result checkbox** is selected (in step 6), only new results from subsequent searches are sent in the alert payload. Otherwise, all results are sent. |
```